.evergreen/utils.sh

```diff
@@ -31,8 +31,7 @@ createvirtualenv () {
 
     python -m pip install --upgrade pip
     python -m pip install --upgrade setuptools wheel
-    # lxml only has wheels for macos 10.15+
-    python -m pip install unittest-xml-reporting || true
```

Comment:
Let's avoid changing this for now since we don't always need tox where createvirtualenv is used. Instead let's install tox in tox.sh.